CMakeLists.txt

```diff
@@ -267,6 +267,7 @@ add_subdirectory(compaction)
 add_subdirectory(cql3)
 add_subdirectory(data_dictionary)
 add_subdirectory(dht)
+add_subdirectory(ent)
```

Comment:
might want to rephrase the commit message so it also explains the EaR support in scylla to posterity .

BTW, IIUC, we are fine with coarse grained port.